advanced-usage/index.md

```diff
@@ -172,7 +173,7 @@ specifies the requirements for a service page to appear in the sidebar
    - Remove existing camera/endpoint/bridges configuration
 - Remove log files from BlueOS services (to reduce space usage on the SD card)
 - Download log files from BlueOS services to report a problem
-  - Old logs are aggregated and kept as zip files
+  - Old logs are aggregated and kept as zip files, and automatically deleted if the space runs out `(New in 1.2)`
```

Comment:
This is a bit imprecise:
1. the system logs are downloaded as a .zip files
2. old logs are aggregated as .gz files